docs/api/download-item.md

```diff
@@ -100,6 +100,8 @@ Returns `Boolean` - Whether the download is paused.
 
 Resumes the download that has been paused.
 
+**Note:** To enable resumable downloads in the narrow sense the server you are downloading from must support range requests and provide a Last-Modified as well as an ETag header value. Otherwise `resume()` will dismiss previously received bytes and restart the download from the beginning. 
```

Comment:
Disclaimer: i'm not a native speaker as you already noticed 😇 
I wanted to stress the difference between a _real resume_ (in a narrow sense) where the download continues where it was paused and a _restart_ (resume in a wider sense).